repoclosure/yum.conf

```diff
@@ -187,9 +187,9 @@ baseurl=https://copr-be.cloud.fedoraproject.org/results/ndp/qpid/epel-6-x86_64/
 name=Pulp Community Release
 baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/stable/latest/6/x86_64
 
-[el6-foreman-client-nightly]
-name=Foreman Client nightly EL6
-baseurl=http://koji.katello.org/releases/yum/foreman-client-nightly/el6/x86_64/
+[el6-foreman-client-3.0]
+name=Foreman Client 3.0 EL6
+baseurl=http://koji.katello.org/releases/yum/foreman-client-3.0/el6/x86_64/
```

Comment:
With the drop of EL6 clients here on this PR for [tool_belt](https://github.com/theforeman/tool_belt/pull/388/), do we still need EL6 clients here?

Reply:
We don't :D  Removing

Reply:
Removed.  Dropped f29 as well for similar reasons